src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -1258,6 +1258,8 @@ void addSubpatternsForTuple(ImmutableArray<TypeWithAnnotations> elementTypes)
                         hasErrors = true;
                     }
                     break;
+                case BinaryOperatorKind.String:
+                case BinaryOperatorKind.Bool:
```

Comment:
What test covers this change? I was looking for a test that would produce `ERR_UnsupportedTypeForRelationalPattern`.

Reply:
`PatternMatchingTests3.Relational_01` (an existing test)

---
In reply to: [400464669](https://github.com/dotnet/roslyn/pull/42313#discussion_r400464669) [](ancestors = 400464669)